src/js/components/DataChart/doc.js

```diff
@@ -87,8 +120,24 @@ export const doc = DataChart => {
       ]),
       PropTypes.string,
     ]).description(`The spacing between the axes and the Charts.`),
+    guide: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        x: PropTypes.shape({
+          granularity: granularityType,
+        }),
+        y: PropTypes.shape({
+          granularity: granularityType,
```

Comment:
What is the granularity and what does it impact on the UI?

Reply:
documentation added